WireAPI/Tests/WireAPITests/APIs/TeamsAPI/TeamsAPITests.swift

```diff
@@ -55,7 +55,7 @@ final class TeamsAPITests: XCTestCase {
 
     func testGetTeamMembersRequest() async throws {
         try await apiSnapshotHelper.verifyRequestForAllAPIVersions { sut in
-            _ = try await sut.getTeamMembers(for: .mockID1, maxResults: 2_000)
+            _ = try await sut.getTeamMembers(for: .mockID1, maxResults: 2000)
```

Comment:
there's another rule that does the exact opposite right ? like formatting 2000 into 2_000, I think it makes sense for big numbers, 1_000_000 is more readable but for small numbers 1000 is fine

Reply:
This is configurable. I'll put you down as in favor but with a different configuration.